spec/mongoid/validatable/uniqueness_spec.rb

```diff
@@ -2472,4 +2472,31 @@ class SpanishActor < EuropeanActor
       }.to raise_error(Mongo::Error::OperationFailure)
     end
   end
+
+  describe "i18n" do
+
+    context 'when using a different locale' do
+
+      around do |example|
+        I18n.with_locale(:fr) { example.run }
+      end
+
+      before do
+        # Translation key location is as per rails-i18n gem.
+        # See: https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/en.yml
+        I18n.backend.store_translations(:fr, { errors: { messages: { taken: 'est déjà utilisé(e)' } } })
+      end
+
+      after do
+        I18n.backend.translations.delete(:fr)
```

Comment:
This needs to `send(:translations)` for i18n 1.0 where `translations` is not public.

Reply:
Done. Please also accept [my other PR]
(https://github.com/mongodb/mongoid/pull/5223) which sets the minimum I18n version to 1.1, it will make everyone's life much easier.